docs/docs/quick_start/advanced_features/volume_eviction.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_position: 3
+sidebar_label: "Eviction"
+---
+
+# Volume Eviction
+
+Volume eviction/migration is one of most important funcions in the HwameiStor, especially in the production environment.
+HwameiStor will keep the data in the volume when migrate it.
+
+Once a Kubernetes node or pod is evicted by the system for any reason, HwameiStor will detect all the volume replicas located on this node, and automatically migrate them to other avaliable nodes. So that, the evicted pods can be rescheduled to another node and continue to run.
+
+## Node Eviction
+
+In a Kubernetes cluster, a node can be drained by using the following procedure. So that, all the pods and volume replicas on this node will be evicted, and then continue the services on other avaliable node.
```

Comment:
```suggestion
In a Kubernetes cluster, a node can be drained by using the following procedure. So that, all the pods and volume replicas on this node will be evicted, and then continue the services on other avaliable nodes.
```